web/ViewModels/LandlordProfileModel.cs

```diff
@@ -14,6 +14,8 @@ public class LandlordProfileModel
     [DisplayName("Title")]
     public string Title { get; set; } = string.Empty;
 
+    public string? TitleInput { get; set; }

```

Comment:
The Title and Other fields won't be filled in correctly when you edit a landlord because they're not mapped directly to the database model anymore.

One way around this would be to keep the server-side code as just having `Title` and not `TitleInput` and then use some Javascript stuff when the form is submitted to put the correct value into `Title`.

Or we could do it this way but then add some extra logic into the the `fromDbModel` to fill `Title` and `TitleInput`.

@oskarwilliams378 what do you think?